esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -972,7 +973,7 @@ bool EspHttpBinding::basicAuth(IEspContext* ctx)
         return false;
     }
 
-    bool authenticated = m_secmgr->authorize(*user, rlist, ctx->querySecureContext());
+    bool authenticated = CSecManagerTraceDecorator(*m_secmgr).authorize(*user, rlist, ctx->querySecureContext());
```

Comment:
The decorator always adds, and releases, its own reference to the object, without affecting ownership of the reference passed to it. I believe it is safe.

As it's being used, linking might not be needed. If the decorator ever has a longer life than one manager call, it will be.

Reply:
Not being an expert with Owned and Linked, but is the reference count for the object shared between the original Owned and the Linked pointers?